sdk/core/azure-json/src/main/java/com/azure/json/JsonOptions.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.json;
+
+/**
+ * Contains configuration options for creating a {@link JsonReader} or {@link JsonWriter}.
+ */
+public final class JsonOptions {
```

Comment:
future concern: we probably should allow implementations to have implementation-specific options and then would need to plumb jsonoptions through clients.

Reply:
Yep! Once we want to allow for implementation specific options we can make this class non-final